### PR TITLE
Create avigoggles_yw.dm

### PR DESCRIPTION
**Pull request by KBraid**
This is the code to import my fluff item into the game, hopefully it works? Been following SplinterGP's instructions.
[aviation_goggles.zip](https://github.com/Yawn-Wider/YWPolarisVore/files/4907844/aviation_goggles.zip)

### CODE REVIEW

**Comment by SplinterGP** [comment]
Try to put it in custom_clothes_yw

**Comment by SplinterGP** [comment]
If you cant i can manage to for you

**Comment by izac112** [comment]
Needs sprite, (unless it uses a base one) and needs to be put in to the loadout file so you can actually get it.

**Comment by SplinterGP** [comment]
Should be good to go now

File: code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm [hunk 1]
```diff
@@ -628,6 +628,13 @@
 
 //  K CKEYS
```
Comment (izac112) [inline code/modules/client/preference_setup/loadout/loadout_fluffitems_yw.dm:630]:
```suggestion
//kbraid
//Eili
```

**Pull request merged**